site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -45,7 +45,8 @@ This is an optional list of domains that this SAML v2 provider should be used fo
 endif::[]
 
 [field]#identityProvider.emailClaim# [type]#[String]# [optional]#Optional#::
-The name of the email claim (Attribute in the `Assertion` element) in the SAML response that FusionAuth uses to uniquely identity the user. If this is not set, the `useNameIdForEmail` flag must be true.
+The name of the email claim in the SAML response that FusionAuth uses to uniquely identity the user.
```

Comment:
Looks like you changed the wording here, we should probably keep the word `Assertion` in here.